q2_quality_control/_threshold_graph/_visualizer.py

```diff
@@ -40,16 +41,31 @@ def _check_inputs(**kwargs):
 
 TEMPLATES = pkg_resources.resource_filename(
     'q2_quality_control._threshold_graph', 'assets')
+_blast_url_template = ("http://www.ncbi.nlm.nih.gov/BLAST/Blast.cgi?"
+                       "ALIGNMENT_VIEW=Pairwise&PROGRAM=blastn&DATABASE"
+                       "=nt&CMD=Put&QUERY=%s")
 
 
-def decontam_score_viz(output_dir, decontam_scores: qiime2.Metadata,
-                       table: pd.DataFrame, threshold: float = 0.1,
+def format_fasta(header, sequence):
+    return f">{header}\n{sequence}\n"
+
+
+def decontam_score_viz(output_dir, decontam_scores: pd.DataFrame,
+                       table: pd.DataFrame,
+                       rep_seqs: pd.Series = None,
+                       threshold: float = 0.1,
                        weighted: bool = True, bin_size: float = 0.02):
     _check_inputs(**locals())
     # initalizes dictionaries for iteration
     table_dict = dict(table)
     decontam_scores_dict = dict(decontam_scores)
 
+    # Sets rep seq flags
+    rep_seq_indicator = ["Are there rep seqs?"]
```

Comment:
Can you explain what you're trying to do with this? I can try to help you figure out a better way.

Reply:
I am attempting to set a flag for rendering on the html side to determine when there are rep_seqs objects passed in or not.